backend/types/emails.py

```diff
@@ -14,7 +14,7 @@ class SingleEmailInput:
     destination: str | list[str]
     subject: str
     content: str | SingleTemplatedEmailContent
-    ConfigurationSetName: str | None = None
```

Comment:
ConfigurationSetName is optional though, so this breaks the actual functionality (type hints anyways)

Reply:
I did changed that because `SESV2Client` expects `ConfigurationSetName` to be only of a type `str` and `mypy` complains about it. So as it need to be kept as `str | None = None` we can set `# type: ignore[arg-type]` or use in *settings/helpers.py* on line 132:
```
ConfigurationSetName=data.ConfigurationSetName or "",
```
insted of just
```
ConfigurationSetName=data.ConfigurationSetName,
```

Reply:
Yeah that works